scrap.js

```diff
@@ -17,21 +19,83 @@ async function fetchLeaders(id) {
   return leaders
 }
 
-async function fetchOrgsWithLeaders() {
+async function searchGitHubOrgs(query) {
+  const token = process.env.GITHUB_TOKEN
+  const res = await fetch(
+    `https://api.github.com/search/users?q=${query}%20type:org`,
+    {
+      headers: token ? { Authorization: `token ${token}` } : {}
+    }
+  )
+  const { items } = await res.json()
+  return items
+}
+
+async function findOrganization({
+  name,
+  description,
+  mailing_list,
+  website_url,
+  irc_channel,
+  blog_url,
+  guide_to_working_url,
+}) {
+  if (overrides[name]) {
+    return overrides[name]
+  }
+
+  const githubPattern = /(?:https?:\/\/)?github\.com\/([a-zA-Z0-9]+)/i
+  const websites = [
+    mailing_list,
+    website_url,
+    irc_channel,
+    blog_url,
+    guide_to_working_url,
+  ]
+
+  const orgFromWebsites = websites
+    .map(website => (githubPattern.exec(website) || [])[1])
+    .find(org => org)
+
+  if (orgFromWebsites) {
+    return orgFromWebsites
+  }
+
+  const orgFromDescription = (githubPattern.exec(description) || [])[1]
+
+  if (orgFromDescription) {
+    return orgFromDescription
+  }
+
+  const removePattern = /the|project|\([a-zA-Z]+\)/gi
+  const searchQuery = name.replace(removePattern, '').trim()
+  const searchResults = await searchGitHubOrgs(searchQuery)
+
+  if (searchResults.length > 0 && searchResults[0].score > 10) {
+    return searchResults[0].login
+  }
+
+  return null
+}
+
+async function fetchOrgsWithData() {
   const orgs = await fetchOrgs()
   const fetchingLeaders = orgs.map(org => fetchLeaders(org.id))
+  const fetchingGitHub = orgs.map(org => findOrganization(org))
   const orgLeaders = await Promise.all(fetchingLeaders)
+  const orgGitHub = await Promise.all(fetchingGitHub)
 
   return orgs.map((org, index) =>
     Object.assign(org, {
       leaders: orgLeaders[index],
+      github: orgGitHub[index],
     })
   )
 }
 
 ;(async () => {
-  const data = await fetchOrgsWithLeaders()
-  
```

Comment:
This line too, please.
You can check with git diff before pushing to make sure that everything is tidy.